core/keyserver/keyserver.go

```diff
@@ -656,7 +659,11 @@ func (s *Server) BatchQueueUserUpdate(ctx context.Context, in *pb.BatchQueueUser
 	tdone()
 
 	// Save mutation to the database.
-	wmLogID, wmTime, err := s.logs.Send(ctx, directory.DirectoryID, in.Updates...)
+	wmLogID, err := s.randLog(ctx, directory.DirectoryID)
```

Comment:
This is a natural place to link to #1377 - you're making a QoS decision here given the current implementation and it's worth noting that.

Reply:
Please address this, otherwise approval.